app/views/UpfrontPaymentSummary.scala.html

```diff
@@ -0,0 +1,112 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+@import requests.RequestSupport
+@import messages.Messages
+@import play.api.mvc.Request
+@import play.twirl.api.Html
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import essttp.rootmodel.AmountInPence
+@import controllers.routes
+@import essttp.rootmodel.UpfrontPaymentAmount
+
+@this(
+        layout: templates.Layout,
+        appConfig: AppConfig,
+        requestSupport: RequestSupport,
+        govukSummaryList: GovukSummaryList,
+    govukButton: GovukButton,
+)
+
+@(
+        upfrontPayment: UpfrontPaymentAmount,
+        remainingAmountToPay: AmountInPence,
+        backUrl: Option[String] = None)(implicit
+        request: Request[_]
+)
+
+@import requestSupport._
+
+
+@title = @{
+    Messages.UpfrontPaymentSummary.`Payment summary`.show
+}
+
+@layout(pageTitle = Some(title), showSignOutLink = false, backUrl = backUrl) {
```

Comment:
shouldn't we be showing the sign out link on every page in the journey once you're logged in?

Reply:
Good spot, it wasn't actually used in the template so removing it